index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0"
+        <title></title>
+        <link rel="stylesheet" href="./index.css">
+    </head>
+    <body>
+<div class="card">
+    <div class="search">
+        <input type="text" placeholder="Enter preferred city" spellcheck="false">
+        <button>
+            <img src="./images/search.png">
+        </button>
+    </div>
+    <div class="error">
+        <p> Invalid city name</p>
+    </div>
+    <div class="weather">
+        <img src="./images/rain.png" class="weather-icon">
+        <h1 class="temperature">22°c</h1>
+        <h2 class="city">Lagos</h2>
+        <div class="details">
+            <div class="first-part">
+                <img src="./images/wind.png">
+            <div>
+                <p class="windy">20km/h</p>
+                <p>Wind Speed</p>
+            </div>
+            </div>
+            <div class="first-part">
+                <img src="./images/humidity.png">
+            <div>
+                <p class="humidity">50%</p>
+                <p>Humidity</p>
+            </div>
+            </div>
+        </div>
+
+    </div>
+</div>
+
+<script>
+const apiKey = "e39e501fcd51e7187f1a879817e82803";
```

Comment:
I know this is a test run but api keys SHOULD not be exposed. We use an env file to store it or better still do a server-to-server communication. That way, the frontend does not get to know what the key is about. DO NOT TRUST YOUR FRONTEND WITH ANYBODY.